openadapt/record.py

```diff
@@ -34,12 +36,34 @@
     "action": True,
     "window": True,
 }
-PLOT_PERFORMANCE = False
+PLOT_PERFORMANCE = config.PLOT_PERFORMANCE
+BYTE_TO_MB = float(2**20)
+NUM_MEMORY_STATS = 3
```

Comment:
Is this just to limit the log output?

Reply:
Yeah, without it every single memory allocation would be displayed, even the tiny insignificant ones. If you want more output I could change it to something like 5 or 10.